test/opt/ccp_test.cpp

```diff
@@ -925,6 +925,133 @@ TEST_F(CCPTest, FoldWithDecoration) {
   SinglePassRunAndMatch<CCPPass>(text, true);
 }
 
+TEST_F(CCPTest, DebugSimpleFoldConstant) {
+  const std::string text = R"(
+; CHECK: [[float1:%\w+]] = OpConstant {{%\w+}} 1
```

Comment:
Can you move the checks next to the lines they are checking?  Much easier to read that way.  OK with that change.

Reply:
done